src/mscorlib/src/System/Decimal.cs

```diff
@@ -140,6 +140,9 @@ public struct Decimal : IFormattable, IComparable, IConvertible, IComparable<Dec
         private int lo;
         private int mid;
 
+        internal uint Low => (uint)lo;
```

Comment:
Can we change Decimal to `[StructLayout(LayoutKind.Explicit)]` like it is done in CoreRT, and define these as regular fields? It would avoid the need to have the `Unsafe.As` below, and make the code here more similar to CoreRT.

Reply:
It might be fine to temporarily have these helpers here, until the rest of the decimal code is ready to be shared. I'm just about to finish tuning decimal rounding and conversion functions for CoreRT, after which decimal might be ready for sharing between CoreCLR and CoreRT.